jsonrpc/dispatcher.go

```diff
@@ -39,23 +39,26 @@ type endpoints struct {
 // Dispatcher handles all json rpc requests by delegating
 // the execution flow to the corresponding service
 type Dispatcher struct {
-	logger        hclog.Logger
-	serviceMap    map[string]*serviceData
-	filterManager *FilterManager
-	endpoints     endpoints
-	chainID       uint64
-	priceLimit    uint64
+	logger           hclog.Logger
+	serviceMap       map[string]*serviceData
+	filterManager    *FilterManager
+	endpoints        endpoints
+	chainID          uint64
+	priceLimit       uint64
+	batchLengthLimit uint64
 }
 
-func newDispatcher(logger hclog.Logger, store JSONRPCStore, chainID uint64, priceLimit uint64) *Dispatcher {
+func newDispatcher(logger hclog.Logger, store JSONRPCStore, chainID uint64, priceLimit uint64,
+	batchLengthLimit uint64, blockRangeLimit uint64) *Dispatcher {
```

Comment:
Could you define an argument per line if you need to define them in 2+ lines like this:

```suggestion
func newDispatcher(
        logger hclog.Logger,
        store JSONRPCStore,
        chainID uint64,
        priceLimit uint64,
	batchLengthLimit uint64,
	blockRangeLimit uint64,
) *Dispatcher {
```